addons/mrp/models/mrp_production.py

```diff
@@ -153,6 +153,15 @@ def _get_default_location_dest_id(self):
                                 readonly=True, states={'confirmed': [('readonly', False)]}, default='1')
 
     @api.multi
+    @api.depends('bom_id.routing_id', 'bom_id.routing_id.operation_ids')
+    def _compute_routing(self):
+        for production in self:
+            if production.bom_id.routing_id.operation_ids:
+                production.routing_id = production.bom_id.routing_id.id
+            else:
+                production.routing_id = False
+
+    @api.multi
```

Comment:
it would be useful to add a testcase that adds a route with no workcenter operations and a production location and check that the created moves have the correaction location destination

also add a route with no workcenter operation and no production location, plan should no nothing, produce should do something